tests/channel-qvm-tests.lisp

```diff
@@ -0,0 +1,30 @@
+(in-package #:qvm-tests)
+
+
+(defun make-test-channel-qvm ()
+  (let* ((p1 (make-noise-pred (match-any-n-qubits 1 0 1) 1 "after"))
```

Comment:
I'm not opposed to the switch to `&rest` arguments for these `match-*` functions in general, but in this case I think it makes it harder to distinguish between the `n` argument and the `&rest qubits` args.

Reply:
Yeah, I was thinking that too. I think maybe I should `&rest` the other `match-*` predicates, and just not this one.